netaddr.go

```diff
@@ -323,3 +325,94 @@ func (p IPPort) TCPAddr() *net.TCPAddr {
 		Zone: zone,
 	}
 }
+
+// IPPrefix is an IP address prefix representing an IP network.
+//
+// The first Bits of IP are specified, the remaining bits match any address.
+// The range of Bits is [0,32] for IPv4 or [0,128] for IPv6.
+type IPPrefix struct {
+	IP   IP
+	Bits uint8
+}
+
+// ParseIPPrefix parses s as an IP address prefix.
+// The string can be in the form "192.168.1.0/24" or "2001::db8::/32".
+func ParseIPPrefix(s string) (IPPrefix, error) {
```

Comment:
I think either the doc comment or the name should include something about CIDR notation.